UnitTests/expected/Storyboards-All-CustomName.swift.out

```diff
@@ -93,6 +111,9 @@ struct XCTStoryboardsScene {
       return vc
     }
   }
+  enum TableViewCell: StoryboardSceneType {
```

Comment:
Feels strange to conform a cell to some protocol named "…SceneType". Maybe it's time to rename that protocol `StoryboardBasedType` instead of `StoryboardSceneType` ?